charts/operator-wandb/values.yaml

```diff
@@ -223,6 +226,12 @@ flat-run-fields-updater:
     repository: wandb/local
     tag: latest
 
+filestream:
+  install: false
+  image:
+    repository: wandb/local-dev
+    tag: 0.63.0-danielpanzella-filestream-local.1
```

Comment:
Is this intentionally checked in hardcoded?

Reply:
yeah, since that branch is the only one that works, we'll clean this up when everything settles on the core side

Reply:
it's overridable at several points.

Reply:
makes sense!